app/javascript/components/pages/admin/user-groups-list/selectors.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const getUserGroups = state => state.get("records").get("user_groups").get("data");
```

Comment:
I think this can be and return `fromJS([])` as default value in case there is no data:
```javascript
state.getIn(["records", "user_groups", "data"], fromJS([]));
```

Reply:
Okay